dev/Sidrah-Madiha/vehicles_dataset_classifer_v1.ipynb

```diff
@@ -0,0 +1,944 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sidra.create_confusion_matrix_class_report(y_test, y_pred)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Conclusion:\n",
+    "Overall I got 78 percent accuaracy which doesn't seem good, next step for today will be to first try to improve this model then I will experiment with other models to see comparative performance of other models on this dataset."
```

Comment:
Looking at the confusion matrix and evaluation metric table, can you expand on your interpretation of this overall accuracy? I notice that the per-class accuracy scores are quite different across the classes.